src/utils/hooks/transaction/utils/fee.ts

```diff
@@ -50,9 +52,8 @@ const getTxFeeSwapData = async (
   // First we construct reverse direction trade to get estimated swap path and amount
   const reverseDirectionTrade = window.bridge.amm.getOptimalTrade(nativeTxFee, feeCurrency, pools);
   if (reverseDirectionTrade === null) {
-    throw new Error(
```

Comment:
Is it safe to remove this error altogether? i.e. if `reverseDirectionTrade === null` is always a result of the trade being too small? If so, feel free to merge :)

Reply:
I think it's fine because he uses recursion to keep finding a suitable trade. Awesome approach @peterslany. I love to see recursion being used.

Reply:
But adding to what @tomjeatt was saying, there is no exit if there is no optimal trade found. Do you think we could limit to something like 20 attemps of find an optimal trade, otherwise we fallback to Governance token? @peterslany

Reply:
It is safe since the assumption is that this function is called only for fee currencies that have trading path existing with native fee currency - this is achieved by only showing those currencies in token fee selector. The case you are describing therefore won't happen.